### ISSUE TITLE
Add checkboxes to remember which articles have been read

### ISSUE THREAD

**Issue by stefanfisk** [superherojs/superherojs#31]
Using :visited doesn't work for people like me, since I tend to skim through stuff quickly and then read it thoroughly later on.

I whipped a quick concept prototype together here, and if there's interest I'd gladly complete it: http://htmlpreview.github.io/?https://github.com/stefanfisk/superherojs/blob/gh-pages/index.html

**Comment by ShaneHudson** [comment]
I am very much in support for this. It is used on http://hackdesign.org/ and has made that site so much more useable and useful.

**Comment by kimjoar** [comment]
@stefanfisk Thanks for the input. Sorry for the laaate response. We've had a bit of a break, but now we've ready to start working more with the site again. I know @olav is working on some ideas for design adjustments, so there might be some upcoming changes. We're still not sure about the best way to handle this.

**Comment by olav** [comment]
Good idea. :) I suppose using localStorage to store checkmarks for read articles would be the simplest solution, as long as we're dealing with a static page.

**Comment by stefanfisk** [comment]
yep, saving the checkbox state per URL is already implemented!
